gigascience/gigadb-website
# fix: resolve issue concerning missing  property missing - hide button

# Pull request for issue:https://github.com/gigascience/gigadb-website/issues/1917

This is a pull request for the following functionalities:

* add $is_no_image property + hide button 'your uploaded Datasets'

## How to test?

Steps to reproduce the behavior:

1. Go to '/adminDataset/admin'
2. Click on 'create Dataset'
3. Fill in the required fields
4. Click on 'create'
5. Switch to the user for whom it was created
6. Click on 'your uploaded Datasets'
7. Click on 'update'
8. See error


## Any issues with implementation?

Describe any problems with your implementation

## Any changes to automated tests?

AdminDatasetImageCest added
